langfuse-node/src/langfuse-node.ts

```diff
@@ -25,9 +26,20 @@ export default class Langfuse extends LangfuseCore {
 
   private options: LangfuseOptions;
 
-  constructor(params: { publicKey: string; secretKey: string } & LangfuseOptions) {
-    const { publicKey, secretKey, ...options } = params;
-    super(params);
+  constructor(params?: { publicKey?: string; secretKey?: string } & LangfuseOptions) {
+    const { publicKey, secretKey, ...options } = utils.configLangfuseSDK(params);
+    if (!secretKey) {
+      throw new Error("secretKey is required");
+    }
+    utils.assert(secretKey, "[Langfuse] secretKey is required for instantiation");
```

Comment:
duplicated assertion

Reply:
Assertion from core of the package. Asserts truthy values. Hence kept it.